scripts/Dockerfile.devel

```diff
@@ -1,10 +1,17 @@
 FROM ceph/ceph:v15
 
-ENV GOPATH=/go \
-	GO111MODULE=on
+
+RUN \
+ mkdir -p /go && \
+ curl https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz | tar xvzf - -C /go --strip-components=1
```

Comment:
Please do not run untar verbose, it clutters the output for no good reason.

Reply:
Sure.. done.